scripts/roles.json

```diff
@@ -3176,7 +3425,7 @@
     ],
     "requiredContextRole": [],
     "requiredOwnedElements": [],
-    "requiredProps": [["aria-level", "2"]],
+    "requiredProps": ["aria-level"],
```

Comment:
Again, did the level 2 default change? 

https://www.w3.org/TR/wai-aria-1.2/#heading

Reply:
Yes: https://rawgit.com/w3c/aria/1d856c402eafbe7823021ff4cb5ca44dc1a9edf7/#heading. It didn't make sense in 1.2 since aria-level was also required. A default for a required parameter is useless.